app/src/main/java/com/onewelcome/showcaseapp/ui/screens/sections/SdkInitializationScreen.kt

```diff
@@ -2,52 +2,154 @@ package com.onewelcome.showcaseapp.ui.screens.sections
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
 import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
 import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Surface
+import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.navigation.NavController
 import com.onewelcome.showcaseapp.R
+import com.onewelcome.showcaseapp.ui.components.ExpandableCard
+import com.onewelcome.showcaseapp.ui.components.NumberSettingTextField
+import com.onewelcome.showcaseapp.ui.components.SettingCheckbox
+import com.onewelcome.showcaseapp.ui.theme.Dimensions
 import com.onewelcome.showcaseapp.viewmodel.SdkInitializationViewModel
+import com.onewelcome.showcaseapp.viewmodel.SdkInitializationViewModel.State
+import com.onewelcome.showcaseapp.viewmodel.SdkInitializationViewModel.UiEvent
 
 @Composable
 fun SdkInitializationScreen(
   navController: NavController,
   viewModel: SdkInitializationViewModel = hiltViewModel()
 ) {
-  SdkInitializationScreenContent(onInitializeSdkClicked = { viewModel.initializeOneginiSdk() })
+  SdkInitializationScreenContent(
+    uiState = viewModel.uiState,
+    onNavigateBack = { navController.popBackStack() },
+    onEvent = { viewModel.onEvent(it) })
 }
 
 @Composable
-private fun SdkInitializationScreenContent(onInitializeSdkClicked: () -> Unit) {
-  Surface(
-    modifier = Modifier.fillMaxSize(),
-    color = MaterialTheme.colorScheme.background
-  ) {
+private fun SdkInitializationScreenContent(
+  uiState: State,
+  onNavigateBack: () -> Unit,
+  onEvent: (UiEvent) -> Unit
+) {
+  Scaffold(
+    topBar = { TopBar(onNavigateBack) },
+  ) { innerPadding ->
     Column(
-      verticalArrangement = Arrangement.Center,
-      horizontalAlignment = Alignment.CenterHorizontally
+      modifier = Modifier
+        .padding(innerPadding)
+        .padding(start = Dimensions.standardPadding, end = Dimensions.standardPadding)
     ) {
+      SettingsSection(
+        modifier =
+          Modifier
+            .weight(1f)
+            .padding(bottom = Dimensions.smallPadding),
+        uiState = uiState,
+        onEvent = onEvent
+      )
       Button(
```

Comment:
WDYT on doing it in similar way as its done on iOS? We could attach the button (sticky button) to the bottom of the page, so it's never hidden. Then, there's no need to scroll to the button if you want to manipulate any options, it's always there to click.

Reply:
done in next PR 😄